.github/workflows/docker_cell-type-ewings.yml

```diff
@@ -40,7 +40,7 @@ jobs:
   test-build:
     name: Test Build Docker Image
     if: github.event_name == 'pull_request' || (contains(github.event_name, 'workflow_') && !inputs.push-ecr)
-    runs-on: ubuntu-latest
+    runs-on: openscpca-22.04-big-disk
```

Comment:
Can you explain to me why we need this? It will be challenging to manage this under our current paradigm.

Reply:
Because I was getting an error running out of disk space when trying to build the image (see https://github.com/AlexsLemonade/OpenScPCA-analysis/actions/runs/11844604285). 

I can see if I can make the image smaller?

Reply:
Or skip the test build for this module... but that seems not great.

Reply:
I've made a change for now, but if we _could_ make this smaller, that'd be swell. I prefer how we were handling permissions.